internal/cmd/root.go

```diff
@@ -131,6 +131,7 @@ func Execute(version, date string) {
 	if err != nil {
 		err := beautifyUnknownAndMissingCommandsError(cmd, err)
 		p.Error(err.Error())
+		p.Debug(print.ErrorLevel, "execute err: %v", err)
```

Comment:
This seems redundant to me as error message printed one line above.

Reply:
But they serve different purposes: 
- `p.Error` is a normal print and is always shown to the user, no matter the `verbosity`
- `p.Debug` is a structured log and is only shown if verbosity is set to `debug` and includes more info such as the timestamp and the call trace

I would say for now we keep both and then if we get feedback that it is redundant (or we have the debug logs too cluttered) we can remove it